### ISSUE TITLE
version 0.4.14 shortcuts order process

### ISSUE THREAD

**Issue by semiosys** [nenes25/eicaptcha#55]
### Issue description
Once eicaptacha is activated, recapatcha is working well but whenever once puts an order, order does not appear in the order list, it is not even stored in database. However the payment go through (For instance paypal payment is debited) but there is no way to discover or guess what the customer has ordered : no trace of the order! Deactivating eicaptacha module solves the issue.

**Comment by metacreo** [comment]
Problem with form selectors. 
contact-form-box - OK
account-creation_form - OK
new_account_form - NOT defined (guest checkout and simple checkout pages)

**Comment by metacreo** [comment]
temp fix:

in eicaptcha.php after line 388 ($prestashop_version = '16';) add code:
```
                $cu_cntrl = Tools::getValue('controller');
				if ($cu_cntrl == 'orderopc') {
					$form_selector = '#new_account_form';
				}
```
in views/js/eicaptcha.js line 26 change js code:
`    $("#submitAccount).live("click", function () {
`
to
`    $("#submitAccount, #submitGuestAccount").live("click", function () {
`
clear all cache, enjoy!

**Comment by metacreo** [comment]
Thank You! For nice module! P/S: Remember that submitGuestAccount and submitAccount can be on one page as in warehouse popular theme. I have several shops of ps 1.6 and all upgraded to latest 1.6.1.18. So... some presta`s not have hook 'header' insted of this have 'displayHeader' and some PS contrariwise. I don't known why but it is so. For example ps_1.6.1.18_default-bootstrap have displayHeader. ps_1.6.1.18_warehouse_3.8.5 have header hook. Maybe it depends on theme. Good luck and thanks again.

**Comment by nenes25** [comment]
Fixed in release 0.4.17
The catcha is disabled on opc checkout.
